### ISSUE TITLE
Update fails because Steam tries to remove nvidia-driver-495

### ISSUE THREAD

**Issue by salihmarangoz** [ValveSoftware/steam-for-linux#8296]
#### Your system information

* Steam client version (build number or date): **Nov 22 2021, at 22:12:42**
* Distribution (e.g. Ubuntu): **Ubuntu 20.04**
* Opted into Steam client beta?: **No**
* Have you checked for system updates?: **Yes. Latest version. "Pre-released updates (focal proposed)" is also checked.**

#### Please describe your issue in as much detail as possible:

A terminal comes up every time the steam is started but the problem never gets solved. 
https://gist.github.com/salihmarangoz/5c6181977dc49cfde3bcff270c5127e0

#### Steps for reproducing this issue:

1. Start Steam
2. A terminal comes up, press enter and enter password.

**Comment by kisak-valve** [comment]
Hello @salihmarangoz, there's a long standing issue with NVIDIA's third party repository for CUDA which is being tracked at #5778. The package repository does not provide 32 bit userspace libraries in the repository while also explicitly asking for the driver version from there and that causes that specific video driver install method incompatible with Steam.

This can not be resolved on Valve's side. Closing in favor of the older issue report.

**Comment by salihmarangoz** [comment]
I was having issues with some games. I was getting 1 fps and nvidia-smi wasn't showing some games. I think everything was being redirected to iGPU even though I applied "prime-select nvidia". I have purged all nvidia related packages then removed CUDA repository ("deb http://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/ /") and installed packages "nvidia-driver-495 libnvidia-gl-495:i386 libnvidia-gl-495". 1 fps problems got solved and steam is not trying to install packages anymore. 
Thank you.
